embedding/csharp/Tizen.Flutter.Embedding/FlutterApplication.cs

```diff
@@ -10,11 +10,36 @@
 
 namespace Tizen.Flutter.Embedding
 {
+    /// <summary>
+    /// Enumeration for the renderer type of the engine.
+    /// </summary>
+    public enum FlutterRendererType
+    {
+        /// <summary>
+        /// The renderer based on EvasGL.
+        /// </summary>
+        EvasGL = 0,
+        /// <summary>
+        /// The renderer based on EGL.
+        /// </summary>
+        EGL
```

Comment:
nit: Just for consistency.
```suggestion
        EGL,
```

Reply:
done